### PR TITLE
Lanth quests

### PR DESCRIPTION
**Pull request by Elisis**
Update multiblock quests to reflect current behaviour

### CODE REVIEW

**Comment by Elisis** [comment]
Wait this branch being old may fuck with the qb
Let me redo

**Comment by Dream-Master** [comment]
check the nuclear changes not that we get merge issues @Elisis

**Comment by Elisis** [comment]
OH
Okay I'll leave this for later then

**Comment by Elisis** [comment]
It looks like it's changing a whole lot of stuff, it might not be dealing with past commits well. I may need to redo it

File: config/betterquesting/DefaultQuests.json [hunk 1]
```diff
@@ -351562,7 +351562,7 @@
       "home_image:8": "dreamcraft:textures/gui/gtnh_title.png",
       "party_enable:1": 1,
       "pack_version:3": 2110,
-      "editMode:1": 0,
+      "editMode:1": 1,
```
Comment (Steelux8) [inline config/betterquesting/DefaultQuests.json:351565]: Keep edit mode off when doing quest changes.

**Review by Steelux8 — CHANGES_REQUESTED** [review]

**Review by Dream-Master — APPROVED** [review]

**Review by Steelux8 — APPROVED** [review]

**Pull request merged**